client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -261,13 +261,46 @@ class OrderDetailsTable extends Component {
 		);
 	};
 
+	renderRefundTotals = () => {
+		const { isEditing, order, translate } = this.props;
+		const refundValue = getOrderRefundTotal( order );
+		if ( isEditing || ! refundValue ) {
+			return null;
+		}
+		const showTax = this.shouldShowTax();
+		const totalValue = getCurrencyFormatDecimal( order.total, order.currency ) + refundValue;
+
+		return (
+			<Fragment>
+				<OrderTotalRow
+					className="order-details__total-refund"
+					currency={ order.currency }
+					label={ translate( 'Refunded' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-order-refund-display)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).